mmd_tools/operators/misc.py

```diff
@@ -50,27 +50,35 @@ def execute(self, context):
         obj = context.active_object
         root = mmd_model.Model.findRoot(obj)
         if root is None:
-            self.report({ 'ERROR' }, 'Select a MMD model') 
-            return { 'CANCELLED' } 
+            # self.report({ 'ERROR' }, 'Select a MMD model') 
+            # return { 'CANCELLED' }
+            # find all meshes in the active layer
+            act_layer = bpy.context.scene.active_layer
+            meshes_list = [o for o in bpy.data.objects
+                           if o.layers[act_layer] and o.type == 'MESH' and o.mmd_type == 'NONE']
+            active_mesh = meshes_list[0]
```

Comment:
I'm not sure this feature is good for this addon or not...(a little risky?!)
If i have a mmd model in layer 1 which is active layer, and i add a cube in the same layer and the cube is the active object. When i hit 'Join Meshes' button, my meshes of the mmd model might be joined to the cube. Is this a case you want ?
Honestly, i think it should be enough to just support for joining meshes of a mmd model. :)